README.md

```diff
@@ -7,23 +7,61 @@
 
 ## Installation
 
-```
+```shell
 curl -sfL http://get.kubearmor.io/ | sudo sh -s -- -b /usr/local/bin
 ```
 
-### Installing from Source 
+### Installing From Source
 
 Build karmor from source if you want to test the latest (pre-release) karmor version.
 
-```
+```shell
 git clone https://github.com/kubearmor/kubearmor-client.git
 cd kubearmor-client
 make install
 ```
 
-## Usage
+### Steps to Verify the Binary (Recommended)
+
+We sign all releases with `cosign`, therefore we recommend verifying **karmor** tarball prior to its installation.
+
+Below are the instructions to verify the binary using `cosign` for version `v1.1.0`.
+
+- Use an env variable for the version
+
+```shell
+export KARMOR_VERSION="v1.1.0"
+```
+
+- Download release tarball, certificate, and signature files
+
+<details>
+  <summary>Details</summary>
+
+```shell
+curl -LO https://github.com/kubearmor/kubearmor-client/releases/download/${KARMOR_VERSION}/karmor_1.1.0_linux_amd64.tar.gz
```

Comment:
```suggestion
curl -LO https://github.com/kubearmor/kubearmor-client/releases/download/${KARMOR_VERSION}/karmor_${KARMOR_VERSION#v}_linux_amd64.tar.gz
```

We can use version in the name as well. Or we can export `KARMOR_VERSION=1.1.0` and use `v${KARMOR_VERSION}`

You can choose either of them.

Reply:
Thanks, this is niuce